### ISSUE TITLE
Files lost - notification says is added to ignored list

### ISSUE THREAD

**Issue by marcheimendinger** [nextcloud/desktop#1691]
### Expected behaviour
The file are not deleted or ignored if we don't tell the client to do it explicitly.

### Actual behaviour
The client ignores some file and, more unpleasingly, deletes some on local disk and reflects this on the server. So the file is completely deleted.

### Steps to reproduce
Wasn't able to reproduce the issue but had this problem on multiple files on two different Nextcloud installations (with the client running always on macOS). The client prompts with a warning message saying the file is in the ignored files list, even if we did nothing to explicitly add it on this list (see the screenshot). Also in the notification the filename is wrong, with some strange `~$` at the beginning. It happens on multiple types of files (.docs, .pdf, etc.).

<img width="883" alt="Capture d'écran 2019-12-19 09 09 43" src="https://user-images.githubusercontent.com/43568598/71270784-b8075480-2352-11ea-8b2e-e948c4802f72.png">

### Client configuration
Client version: 2.5.1final

Operating system: macOS 10.14.5

OS language: French

Installation path of client: /Applications

### Server configuration
Nextcloud version: 16.0.1

**Comment by doits** [comment]
I had similar issues where the mac client deleted local files that weren't synced and so I nearly lost them (worst thing the client can do IMO but I could restore them from another location). But after upgrading to 2.6.2.rc1 it is running fine and did not do it anymore (yet). I think before it happened for me because of sync errors (versions < 2.6.2 were really unstable and had a lot of problems for me).

Bottom line: Maybe upgrading to 2.6.2rc1 will prevent that issue. Otherwise I think the Nextcloud team needs a reliable way to reproduce the issue to fix it, or else it is just a guessing game. But providing those reproduce steps can be hard, I know it ...

**Comment by marcheimendinger** [comment]
Will try, thanks. Happy Holidays !

**Comment by camilasan** [comment]
Hi, Could you try to run the most recent desktop client and server to check if you still have this issue? The latest version can be seen by checking [https://nextcloud.com/install/#install-clients](https://nextcloud.com/install/#install-clients) and [https://nextcloud.com/install/#instructions-server](https://nextcloud.com/install/#instructions-server) respectively.

If you still have this problem with the latest desktop client, please re-open the issue.

Thanks!
